packages/iov-keycontrol/src/hdpaths.ts

```diff
@@ -1,15 +1,28 @@
 import { Slip10RawIndex } from "@iov/crypto";
 
 export class HdPaths {
+  public static readonly purposes = {
```

Comment:
why public?

Reply:
Mostly so we don't have to type out the faucet number, but maybe this shouldn't be in the class anyway.

Reply:
But `private` would do the job as well if there is a faucet path helper